presto-main/src/main/java/com/facebook/presto/operator/HashBuilderOperator.java

```diff
@@ -225,7 +226,7 @@ private int getAndIncrementPartitionIndex(Lifespan lifespan)
 
     private Optional<Runnable> finishMemoryRevoke = Optional.empty();
 
-    public HashBuilderOperator(
+    private HashBuilderOperator(
```

Comment:
ditto

Reply:
sure. My idea was that since I'm dealing with hashbuilderoperator, this counted as within scope, but also I don't mind not doing it in this commit